addon/ng2/models/webpack-build-production.ts

```diff
@@ -15,7 +15,6 @@ export const getWebpackProdConfigPartial = function(projectRoot: string, appConf
     },
     plugins: [
       new WebpackMd5Hash(),
-      new webpack.optimize.DedupePlugin(),
       new webpack.optimize.UglifyJsPlugin(<any>{
         mangle: { screw_ie8 : true, keep_fnames: true },
```

Comment:
I'm not aware of that discussion, I was just taking a crack at updating. There's still an issue with e2e tests, but I wanted to get something pushed up for others to review.